middleware/index.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express'
+import path from 'path'
+
+module.exports = ({middlewarePath='/falcor-postman', app}) => {
+  app.use(express.static(path.join(__dirname, '/../dist')))
+
+  return (req, res, next) => {
+    if (req.url === middlewarePath) {
+      res.status(200)
+      res.send(path.join(__dirname, '/../dist/fp.html'));
```

Comment:
Thats only a development workflow. 
'webpackServeBundle' allows to run the example in dev mode (`isDevelopment`) serving in that case the bundle via `webpack-dev-middleware` and `webpack-hot-middleware`, thats why is in there.